test/unit/ngsiv2/config-test.js

```diff
@@ -43,7 +43,7 @@ config.amqp = {
 };
 
 config.iota = {
-    logLevel: 'FATAL',
+    logLevel: 'DEBUG',
```

Comment:
Before merging this PR, the log level must be restored to `FATAL`.

Reply:
Fixes in 375d69e